python/test/t_MaximumLikelihoodFactory_std.py

```diff
@@ -15,3 +15,45 @@
 inf_distribution = factory.build(sample)
 print(inf_distribution)
 
+class TruncatedFactory(object):
+    def __init__(self, distribution, lb=None, ub=None):
```

Comment:
@jschueller why did you remove it? The problem is that we should set an inequality constraint lb < ub + constrains if parameters should be in [lb, ub]